src/utils/__tests__/geThemeObject.spec.js

```diff
@@ -0,0 +1,23 @@
+import { getThemeObject } from "..";
+import { themes } from "../../theme";
+
+describe("getThemeObject", () => {
+  it("should return global theme when themeName is not present in themes and customTheme is not passed", () => {
```

Comment:
Could you please add a test case for when the `themeName` is invalid and a `customTheme` is passed? Thanks!